misc/python/materialize/checks/all_checks/schemas.py

```diff
@@ -44,7 +44,7 @@ def validate(self) -> Testdrive:
 
 class RenameSchemas(Check):
     def _can_run(self, e: Executor) -> bool:
-        return self.base_version >= MzVersion.parse("0.73.0-dev")
+        return self.base_version >= MzVersion.parse("0.74.0")
```

Comment:
I think it needs to be 0.74.0-dev here, so that the current "main" will run the test. -dev I think sorts before 0.74.0

Reply:
Correct

Reply:
Okay, thanks!